users/api/requests.go

```diff
@@ -17,6 +16,9 @@ type userReq struct {
 }
 
 func (req userReq) validate() error {
+	if !passRegex.MatchString(req.user.Password) {
+		return users.ErrMalformedEntity
```

Comment:
maybe the error should state exactly what the problem is `password doesn't match policy`, something like that